rsmtool/analyzer.py

```diff
@@ -1198,6 +1199,17 @@ def compute_metrics_by_group(self,
                               for col in df_test.columns if col not in ['spkitemid',
                                                                         grouping_variable]}
 
+        # check if any of the standard deviations is zero and
+        # tell user to expect to see many warnings.
+        zero_sd_scores = [score for (score, sd) in population_sd_dict.items() if
+                          np.isclose(sd, 0, atol=1e-07)]
+        if len(zero_sd_scores) > 0:
+            warnings.warn("The standard deviation for {} scores "
+                          "is zero (all scores are the same). You "
+                          "will see multiple warnings about DSM computation "
+                          "since this metrics is computed separately for "
+                          "each subgroup.".format(', '.join(zero_sd_scores)))
+
```

Comment:
```suggestion
            warnings.warn("The standard deviation for {} scores "
                          "is zero (all values are the same). You "
                          "will see multiple warnings about DSM computation "
                          "since this metric is computed separately for "
                          "each subgroup.".format(', '.join(zero_sd_scores)))

```